package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@livechat/eslint-plugin",
+  "version": "0.1.7",
```

Comment:
Why such a strange version? Shouldn't we start with 1.0.0?

Reply:
it's my working version, I will replace it with 1.0.0 when task is ready to merge